internal/support/scheduler/schedule.go

```diff
@@ -498,6 +490,29 @@ func execute(context *IntervalContext, wg *sync.WaitGroup) error {
 	return nil
 }
 
+// EFC We may need to modify this for authorization type in the future
+func getRequestMethod(httpMethod string, executingUrl string, intervalAction contract.IntervalAction) (*http.Request, error) {
```

Comment:
@xmlviking I tested on my local machine and work fine.
Why not use `getHttpRequest` instead of the `getRequestMethod`?